bugbug/model.py

```diff
@@ -340,7 +354,239 @@ def get_labels(self) -> Tuple[Dict[Any, Any], List[Any]]:
         """Subclasses implement their own function to gather labels."""
         pass
 
-    def train(self, importance_cutoff=0.15, limit=None):
+    def train_with_gridserach(self):
```

Comment:
I like the idea of adding a method to do grid search, but it should be part of a separate PR.
Please remove all the grid search related code, and let's add it with a separate PR (and branch).